src/Client.php

```diff
@@ -3,6 +3,7 @@
 
 use DateTime;
 
+const VERSION = "1.1.0";
```

Comment:
should we make this also 1.1.0-dev? And flip it to non-dev during release?

Reply:
yeah I think that'd be fine.